crates/holochain/tests/tests/conductor/mod.rs

```diff
@@ -0,0 +1,341 @@
+use std::collections::HashMap;
+use std::path::PathBuf;
+
+use ::fixt::prelude::*;
+use holochain::sweettest::*;
+use holochain_conductor_api::{AppInfoStatus, CellInfo};
+use holochain_types::prelude::*;
+use holochain_wasm_test_utils::TestWasm;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn can_install_app_with_custom_modifiers_correctly() {
+    let conductor = SweetConductor::from_standard_config().await;
+
+    let (dna, _, _) = SweetDnaFile::unique_from_test_wasms(vec![TestWasm::Create]).await;
+    let path = PathBuf::from(format!("{}", dna.dna_hash()));
```

Comment:
How does this work? I'd expect this to need to be a path to `../../target/wasm32-...`

Reply:
I don't know really, I just copied that part from another test :sweat_smile:

Reply:
It would be good to understand what's going on here. Normally we install apps differently to this so we don't see the paths and they're just found through the wasm test utils logic but that shouldn't work through the admin interface.

I'm wondering whether there is some test code available that shouldn't be. If so, we should be opening an issue about that and this test should be modifying the sweettest interface for installing apps to include a properties override. Because there's quite a lot of boilerplate in preparing a bundle, we'd like to have that contained in sweettest. Ideally, we do those updates as we go, so conductor interface changes come with sweettest changes

Reply:
I see that same code pattern being used in sweettest itself here: https://github.com/holochain/holochain/blob/18b3dc3fe41071a615b7b6629ef6fdcd3793ef3f/crates/holochain/src/sweettest/sweet_app_installation.rs#L19

And also all tests that are defining a custom dna manifest seem to be using this patter, e.g. all tests in [this file](https://github.com/holochain/holochain/blob/18b3dc3fe41071a615b7b6629ef6fdcd3793ef3f/crates/holochain/src/conductor/tests/install_app_bundle.rs#L24).

Reply:
It's only the relative path within the bundle that's being defined there as I understand it.